python-wrapper/src/neo4j_viz/pandas.py

```diff
@@ -12,22 +12,12 @@
 DFS_TYPE = Union[DataFrame, Iterable[DataFrame]]
 
 
-def from_dfs(
-    node_dfs: DFS_TYPE, rel_dfs: DFS_TYPE, node_radius_min_max: Optional[tuple[float, float]] = (3, 60)
+def _from_dfs(
```

Comment:
why the 2 methods now? the signature looks the same?

Reply:
No they're not exactly the same. See the `rename_properties` arg, and how the `from_gds` loader uses it